api/app/utils/fba_calculator.py

```diff
@@ -135,8 +143,30 @@ def calculate_fire_behavour_advisory(station: FBACalculatorWeatherStation) -> Fi
 
     fire_type = get_fire_type(fuel_type=station.fuel_type, crown_fraction_burned=cfb)
     flame_length = get_approx_flame_length(hfi)
-    sixty_minute_fire_size = get_60_minutes_fire_size(lb_ratio, ros)
-    thirty_minute_fire_size = get_30_minutes_fire_size(lb_ratio, ros)
+
+    wind_azimuth = cffdrs.correct_wind_azimuth(station.wind_direction)
+    slope_azimuth = None  # a.k.a. SAZ
+    ground_slope = 0  # right now we're not taking slope into account
+    wsv = cffdrs.calculate_net_effective_windspeed(fuel_type=station.fuel_type, ffmc=station.ffmc,
+                                                   bui=station.bui, ws=station.wind_speed, waz=wind_azimuth,
+                                                   gs=ground_slope,
+                                                   saz=slope_azimuth, fmc=fmc, sfc=sfc,
+                                                   pc=station.percentage_conifer,
+                                                   cc=station.grass_cure,
+                                                   pdf=station.percentage_dead_balsam_fir,
+                                                   cbh=station.crown_base_height,
+                                                   isi=station.isi)
+
+    bros = cffdrs.back_rate_of_spread(fuel_type=station.fuel_type, ffmc=station.ffmc, bui=station.bui,
+                                      wsv=wsv,
+                                      fmc=fmc, sfc=sfc,
+                                      pc=station.percentage_conifer,
+                                      cc=station.grass_cure,
+                                      pdf=station.percentage_dead_balsam_fir,
+                                      cbh=station.crown_base_height)
+
+    sixty_minute_fire_size = get_fire_size(station.fuel_type, ros, bros, 60, cfb, lb_ratio)
```

Comment:
Does this duplicate `get_30_minutes_fire_size` and `get_60_minutes_fire_size`? Do we need all these fire size functions?

Reply:
It does - those ones use the spreadsheet method - I'll delete them on the next round. I've added some comments to them that I'd like in our commit history.

Reply:
I've added comments to that effect.